server/server/src/logging.rs

```diff
@@ -1,39 +1,35 @@
 use std::env;
 
-use fast_log::{
-    consts::LogSize,
-    plugin::{file_split::RollingType, packer::LogPacker},
-    Config as LogConfig,
-};
-use log::LevelFilter;
 use service::settings::{Level, LogMode, LoggingSettings};
+use simple_log::LogConfigBuilder;
 
-pub fn logging_init(
-    settings: Option<LoggingSettings>,
-    apply_config: Option<Box<dyn Fn(LogConfig) -> LogConfig>>,
-    level: Option<Level>,
-) {
+pub fn logging_init(settings: Option<LoggingSettings>, level: Option<Level>, update: bool) {
```

Comment:
Would just add a quick comment somewhere in this method that simple-log is based on log4rs, which we can use if we need to extend logging functionality beyond what's available for simple-log

Reply:
[cbd8bf8](https://github.com/openmsupply/open-msupply/pull/1921/commits/cbd8bf8502926e3be1be0cf5161f62a0590869a7)